packages/@react-aria/test-utils/src/combobox.ts

```diff
@@ -0,0 +1,193 @@
+      if (triggerBehavior !== 'focus') {
+        await this.user.keyboard('{ArrowDown}');
+      }
+    } else if (interactionType === 'touch') {
+      if (triggerBehavior === 'focus') {
+        await this.user.pointer({target: combobox, keys: '[TouchA]'});
+      } else {
+        await this.user.pointer({target: trigger, keys: '[TouchA]'});
+      }
+    }
+
+    await waitFor(() => {
+      if (!isDisabled && combobox.getAttribute('aria-controls') == null) {
+        throw new Error('No aria-controls found on combobox trigger element.');
+      } else {
+        return true;
+      }
+    });
+    let listBoxId = combobox.getAttribute('aria-controls');
+    await waitFor(() => {
+      if (!isDisabled && (!listBoxId || document.getElementById(listBoxId) == null)) {
+        throw new Error(`Listbox with id of ${listBoxId} not found in document.`);
+      } else {
+        return true;
+      }
+    });
+  };
+
+  selectOption = async (opts: {option?: HTMLElement, optionText?: string, triggerBehavior?: 'focus' | 'manual', interactionType?: UserOpts['interactionType']} = {}) => {
+    let {optionText, option, triggerBehavior, interactionType = this._interactionType} = opts;
+    if (!this.combobox.getAttribute('aria-controls')) {
+      await this.open({triggerBehavior});
+    }
+
+    let listbox = this.listbox;
+    if (listbox) {
+      if (!option && optionText) {
+        option = within(listbox).getByText(optionText);
+      }
+
+      // TODO: keyboard method of selecting the the option is a bit tricky unless I simply simulate the user pressing the down arrow
+      // the required amount of times to reach the option. For now just click the option even in keyboard mode
+      if (interactionType === 'mouse' || interactionType === 'keyboard') {
+        await this.user.click(option);
+      } else {
+        await this.user.pointer({target: option, keys: '[TouchA]'});
+      }
+
+      if (option && option.getAttribute('href') == null) {
+        await waitFor(() => {
+          if (document.contains(listbox)) {
+            throw new Error('Expected listbox element to not be in the document after selecting an option');
+          } else {
+            return true;
+          }
+        });
+      }
+    } else {
+      throw new Error("Attempted to select a option in the combobox, but the listbox wasn't found.");
+    }
+  };
+
+  close = async () => {
+    let listbox = this.listbox;
+    if (listbox) {
+      act(() => this.combobox.focus());
+      await this.user.keyboard('[Escape]');
+
+      await waitFor(() => {
+        if (document.contains(listbox)) {
+          throw new Error('Expected listbox element to not be in the document after selecting an option');
+        } else {
+          return true;
+        }
+      });
+    }
+  };
+
+  get combobox() {
+    if (!this._combobox) {
+      throw new Error('Combobox input element hasn\'t been set yet. Did you call `setElement()` yet?');
```

Comment:
get rid of this error since we don't have setElement any more as part of the api